src/components/TitleItem.vue

```diff
@@ -0,0 +1,20 @@
+<template>
```

Comment:
File name should be representative of its purpose. Personally I would say that PageTitle is more fitting for this component.